packages/react/test/reactrouterv5.test.tsx

```diff
@@ -1,4 +1,4 @@
-import { render } from '@testing-library/react';
+import { render, act } from '@testing-library/react';
```

Comment:
You need to change this to { act, render }

Reply:
Fixed with 0e78d1b72d2c040823ac16db1821d3c5473945a3, thanks!